### PR TITLE
moves sso, labels and nodes to setup

### PR DESCRIPTION
**Pull request by klizhentas**

### CODE REVIEW

File: docs/pages/setup/admin/adding-nodes.mdx [hunk 1]
```diff
@@ -0,0 +1,260 @@
+---
+title: Adding Nodes to the cluster
+description: Adding Nodes to the cluster
+---
+
+This guide explains how to label nodes with Open Source, Enterprise Teleport,
```
Comment (zmb3) [inline docs/pages/setup/admin/adding-nodes.mdx:6]: Looks like a copy paste from the labeling nodes guide. Don't we mean to say "this guide explains how to add nodes to a teleport cluster?"

File: docs/pages/setup/admin/adding-nodes.mdx [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+
+```code
+# Generate a short-lived invitation token for a new node:
+$ tctl nodes add --ttl=5m --roles=node,app
+# The invite token: abcdefgh-do-not-use-this-token-123
+
+# You can also list all generated non-expired tokens:
+$ tctl tokens ls
+# Token                            Type            Expiry Time
+# ---------------                  -----------     ---------------
```
Comment (zmb3) [inline docs/pages/setup/admin/adding-nodes.mdx:67]: nit: the token value is long enough that it spills into the type column

File: docs/pages/setup/deployments/aws-terraform.mdx [hunk 3]
```diff
@@ -753,7 +753,7 @@ auth_servers:
 ### Joining nodes via Teleport IoT/node tunnelling
```
Comment (zmb3) [inline docs/pages/setup/deployments/aws-terraform.mdx:753]:
```suggestion
### Joining nodes via Teleport IoT/node tunneling
```

File: docs/pages/setup/deployments/aws-terraform.mdx [hunk 4]
```diff
@@ -753,7 +753,7 @@ auth_servers:
...
 
 To join Teleport nodes from outside the same VPC, you will either need to investigate VPC peering/gateways (out of scope
-for this document) or join your nodes using [Teleport's node tunnelling](../../admin-guide.mdx#adding-a-node-located-behind-nat) functionality.
+for this document) or join your nodes using [Teleport's node tunnelling](../admin/adding-nodes.mdx#adding-a-node-located-behind-nat) functionality.
```
Comment (zmb3) [inline docs/pages/setup/deployments/aws-terraform.mdx:756]:
```suggestion
for this document) or join your nodes using [Teleport's node tunneling](../admin/adding-nodes.mdx#adding-a-node-located-behind-nat) functionality.
```

**Review by zmb3 — CHANGES_REQUESTED** [review]
Found a couple little things to fix.

**Review by zmb3 — APPROVED** [review]

**Comment by klizhentas** [comment]
observing strange behavior with CI/CD, going to reopen

**Pull request closed without merging**